qudipy/io/__init__.py

```diff
@@ -0,0 +1,5 @@
+'''
+Code for handling processing of potential files (generated from nextnano++
+software)
+'''
+from . import dictionary_to_json_conversion 
```

Comment:
Should be renamed together with the file

```suggestion
from . import nextnano
```